web/cobrands/fixmystreet/offline.js

```diff
@@ -289,37 +261,29 @@ fixmystreet.offline = (function() {
     }
 
     // Remove a list of reports from the offline cache
-    function removeReports(urls) {
+    async function removeReports(urls) {
         var pathsRemoved = [];
```

Comment:
Not part of this PR, admittedly, but this variable doesn't seem to be used anywhere.